src/single-spa-angular1.js

```diff
@@ -68,7 +68,9 @@ function mount() {
 
 function unmount() {
 	return new Promise((resolve, reject) => {
-		let rootScope = angular.injector(['ng']).get('$rootScope');
+		let rootElement = angular.element(getContainerEl().childNodes[0]);
```

Comment:
And if so, does angular put that ng-scope attribute on both in debug mode and production mode??